gammapy/data/counts_spectrum.py

```diff
@@ -0,0 +1,188 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (print_function)
+
+import numpy as np
+from astropy import log
+from ..spectrum.energy import *
+import datetime
+from astropy.io import fits
+
+__all__ = ['CountsSpectrum']
+
+class CountsSpectrum(object):
+    """Counts spectrum dataset
+
+    Parameters
+    ----------
+
+    counts: `~numpy.array`, list
+        Counts
+    energy: `~gammapy.spectrum.Energy`, `~gammapy.spectrum.EnergyBounds`.
+        Energy axis
+
+    Examples
+    --------
+
+    .. code-block:: python
+    
+        from gammapy.spectrum import *
```

Comment:
Same here ... also for end-users it's better to never user `import *`.
If you show them where things are in the examples, they'll learn.
Plus `import *` can lead to name conflicts and errors that are hard to understand for novices.